app/models/user.rb

```diff
@@ -0,0 +1,30 @@
+class User < ActiveRecord::Base
+
+  has_many :pictures
+  has_many :comments
+  has_many :likes
+  has_many :liked_pictures, through: :likes, source: :pictures
+  has_many :commented_pictures, through: :comments, source: :pictures
+
+  validates :user_name, presence: true
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable,
+         :omniauthable, :omniauth_providers => [:facebook]
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)